vraptor-core/src/main/java/br/com/caelum/vraptor/converter/DateConverter.java

```diff
@@ -61,16 +61,18 @@ public Date convert(String value, Class<? extends Date> type) {
 		if (isNullOrEmpty(value)) {
 			return null;
 		}
-
-		try {
-			return getDateFormat().parse(value);
-
-		} catch (ParseException pe) {
-			throw new ConversionException(new ConversionMessage(INVALID_MESSAGE_KEY, value));
+		
+		DateFormat formatDateTime = DateFormat.getDateTimeInstance(DateFormat.MEDIUM, DateFormat.MEDIUM, locale);  
+		try {  
+			return formatDateTime.parse(value);  
+		} catch (ParseException pe) {  
+				DateFormat formatDate = DateFormat.getDateInstance(DateFormat.SHORT, locale);  
+			try {  
+				return formatDate.parse(value);  
+			} catch (ParseException pe1) {  
+				throw new ConversionException(new ConversionMessage(INVALID_MESSAGE_KEY, value));
+			}
 		}
 	}
 
-	protected DateFormat getDateFormat() {
```

Comment:
we can't just remove this method, there is apps/users especializing this class and 
overriding this method to change `DateFormat`. If we drop it, we'll break them all